app/main.py

```diff
@@ -2,0 +1,10 @@
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "cp":
+        print("Invaild command format")
+        return
+    _, source, destination = parts
+    if source == destination:
+        return
+    with open(source, "r") as file_in, open(destination, "w") as file_out:
```

Comment:
The function does not handle exceptions that may occur during file operations, such as `FileNotFoundError` if the source file does not exist or `PermissionError` if there are insufficient permissions to read or write the files. Consider adding exception handling to manage these cases gracefully.